cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -482,9 +482,9 @@ class SharedStorageSchema(BaseSchema):
         validate=validate.OneOf(["Ebs", FSX_LUSTRE, FSX_OPENZFS, FSX_ONTAP, "Efs"]),
         metadata={"update_policy": UpdatePolicy.UNSUPPORTED},
     )
-    ebs_settings = fields.Nested(EbsSettingsSchema, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
-    efs_settings = fields.Nested(EfsSettingsSchema, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
-    fsx_lustre_settings = fields.Nested(FsxLustreSettingsSchema, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
+    ebs_settings = fields.Nested(EbsSettingsSchema, metadata={"update_policy": UpdatePolicy.IGNORED})
```

Comment:
It would be good to add a unit test to verify that this new update path is supported. I mean from:
```
  - MountDir: /opt/shared/efs/managed/1
    Name: shared-efs-managed-1
    StorageType: Efs
```
to:
```
  - MountDir: /opt/shared/efs/managed/1
    Name: shared-efs-managed-1
    StorageType: Efs
    EfsSettings:
      DeletionPolicy: Retain|Delete
```

Reply:
Agree. Added